contents/performance/WebPerformance.md

```diff
@@ -0,0 +1,80 @@
+
+Here are several ways how progressive loading can be effectively applied:
+- Load only the [most critical content](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/) as part of the initial render, and fetch remaining resources asynchronously after the render. [Code splitting](https://developers.google.com/web/fundamentals/performance/optimizing-javascript/code-splitting/) can also be helpful here to reduce the amount of Javascript that needs to be loaded for the initial render.
+- Use [lazy loading](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/) to defer requesting non-critical resources until they are needed.
+- Show [content placeholders](https://medium.com/@praveencnaik/content-placeholder-the-new-design-trend-for-audience-involvement-e2ab533d7304) while fetching resources to indicate loading progress to the user.
+
+## Measuring Web Performance
+
+Improving web performance is a continuous, ongoing process. The ability to measure and track performance is necessary for developers to monitor the impact of their changes over time.
+
+Here are some ways that you can measure the performance of your web applications:
+- Test your site under various network conditions with [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/network/network-conditions). This is helpful to get a feel for how the site feels for users on slower devices during the development process.
+- Audit web performance using [Lighthouse](https://developers.google.com/web/tools/lighthouse/). This provides many useful metrics for understanding the performance of your site and identifying performance bottlenecks under controlled conditions.
+- Collect [user-centric performance metrics](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics). This helps you to monitor how your application actually performs in a real-world scenario, and how your site performance relates to other user engagement metrics.
+- Use [WebPageTest](https://www.webpagetest.org/) to test the performance of your website from multiple locations around the world at real consumer connection speeds. This provides real-world performance indicators that is especially useful if the target audience of your site resides in a different region.
+
+
+## Additional Resources
+- [Google's Web Performance Fundamentals](https://developers.google.com/web/fundamentals/performance/why-performance-matters/)
+- [Kinsta's Beginner Guide to Website Speed Optimization](https://kinsta.com/learn/page-speed/)
+- [Twitter Lite Case Studey](https://medium.com/@paularmstrong/twitter-lite-and-high-performance-react-progressive-web-apps-at-scale-d28a00e780a3)
```

Comment:
Can give your commentary about each resource as additional guidance to the reader.